thrust/thrust/iterator/zip_iterator.h

```diff
@@ -296,6 +305,13 @@ class _CCCL_DECLSPEC_EMPTY_BASES zip_iterator : public detail::make_zip_iterator
   //! \endcond
 };
 
+template <typename... Iterators>
+_CCCL_HOST_DEVICE zip_iterator(_CUDA_VSTD::tuple<Iterators...>) -> zip_iterator<_CUDA_VSTD::tuple<Iterators...>>;
```

Comment:
I believe we should also add a guide for thrust::tuple
```suggestion
template <typename... Iterators>
_CCCL_HOST_DEVICE zip_iterator(_CUDA_VSTD::tuple<Iterators...>) -> zip_iterator<_CUDA_VSTD::tuple<Iterators...>>;

template <typename... Iterators>
_CCCL_HOST_DEVICE zip_iterator(THRUST_NS_QUALIFIER::tuple<Iterators...>) -> zip_iterator<_CUDA_VSTD::tuple<Iterators...>>;
```

Reply:
Thrust tuple is:
```c++
namespace thrust {
  using ::cuda::std::thrust;
}
```
So it's the same type.

Reply:
🤦 I live in the past ^^